example-Dockerfile

```diff
@@ -1,28 +1,12 @@
 # better to stick with lts
```

Comment:
Please rename this filename to Dockerfile.

Reply:
ok